gravitee-node-cache/gravitee-node-cache-common/src/main/java/io/gravitee/node/plugin/cache/common/InMemoryCache.java

```diff
@@ -224,12 +224,12 @@ public void putAll(final Map<? extends K, ? extends V> values) {
     }
 
     @Override
-    public V computeIfAbsent(final K key, final Function<? super K, ? extends V> mappingFunction) {
+    public V computeIfAbsent(final K key, final Function<? super K, ? extends V> remappingFunction) {
         this.internalCache.asMap()
             .computeIfAbsent(
                 key,
                 k -> {
-                    V applied = mappingFunction.apply(k);
+                    V applied = remappingFunction.apply(k);
                     notifyListeners(k, applied, null);
                     return new ExpiringValue<>(applied, 0);
```

Comment:
It seems this method always return a expiring value even if the mapping function returns null; As you change the signature of rxComputeIfAbsent to a maybe, a Maybe is not necessary except you test the applied value and return null if null.
The same logic applies to all compute methods.

```suggestion
                    return applied == null ? null : new ExpiringValue<>(applied, 0);
```